tests/h/tasks/indexer_test.py

```diff
@@ -62,16 +61,9 @@ def test_it(self, search_index):
         search_index.sync.assert_called_once_with("test_queue")
 
 
-class TestReportSyncAnnotationsQueueLength:
-    def test_it(self, newrelic, search_index):
-        indexer.report_sync_annotations_queue_length()
-
-        search_index._queue.count.assert_called_once_with(
-            ["storage.create_annotation", "storage.update_annotation"]
-        )
-        newrelic.agent.record_custom_metric.assert_called_once_with(
-            Any.string(), search_index._queue.count.return_value
-        )
+class TestReportJobQueueMetrics:
+    def test_it(self):
+        indexer.report_job_queue_metrics()
```

Comment:
Hmm, I tend to agree, but it goes against the grain of recent discussions about logging. If we were going to go proper nuts about checking logs, it seems odd not to do something similar for actual metrics. Seems inconsistent.

Reply:
I think the difference is that the `Queue.sync()` task actually contained logic and different paths, and needed to log the right messages under the right circumstances. This method is near declarative and tests would just mirror the code in a non-useful way.

I don't think we should have tests for log messages whenever we log anything, FYI. I just wanted tests for `Queue`'s log messages because I felt those were important

Reply:
I think we're coming at the same idea from very slightly different angles. I think the content of the logging was important, and good to check (which is pretty much where we are now), and that separately the passing of the value to logging is less interesting, which is kind of analogous to this type of thing. Assuming the universe still works, it's probably going to log.

So I guess I was trying to take one thing, which is interesting as a whole and separate the interesting and non-interesting parts.

Anyway, I think this is fine. I'm not a huge fan of tests that end up, line for line, replicating the code. It makes a tester feel nice, but make refactoring horrible.